.evergreen.yml

```
@@ -80,6 +80,7 @@ post:
       local_files_include_filter:
         - src/packages/compass-e2e-tests/.log/**/*.log
         - src/packages/compass-e2e-tests/.log/**/*.json
+        - src/packages/compass-e2e-tests/.log/**/*.png
```

Just to match gh config and to avoid missing something in these uploads in the future

```suggestion
        - src/packages/compass-e2e-tests/.log/**/*
```

I'll close this one https://github.com/mongodb-js/compass/pull/2825